v22.2/restore.md

```diff
@@ -159,12 +159,13 @@ Also, a full cluster restore will:
 
 - Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
 - Drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- When the cluster is in a mixed-version state during an [upgrade](upgrade-cockroach-version.html), a full cluster restore will fail. To perform a full cluster restore, it is necessary to [finalize the upgrade](upgrade-cockroach-version.html#step-3-decide-how-the-upgrade-will-be-finalized).
```

Comment:
nit: think we just need `first` in here somewhere, maybe:

> To perform a full cluster restore, it is necessary to first [finalize the upgrade].

or maybe:

> to [finalize the upgrade] beforehand.

Up to you!

Reply:
Done